core/storage/collection/gae_models.py

```diff
@@ -280,21 +280,6 @@ def has_reference_to_user_id(cls, user_id):
         Returns:
             bool. Whether any models refer to the given user ID.
         """
-        more_results = True
```

Comment:
Can you explain more what's going on here? I don't see a corresponding addition to this removal.

Also, the docstring for this method should be changed. Ditto for other methods.

Reply:
This was removed in order to make `GaeIdNotInModelsVerificationJob` work, I will later add the one-job that will aggregate all the user_ids used in all the snapshots and then add a new field in the `CollectionRightsModel` that will contain the IDs so that we don't need to enumerate through all the models when checking `CollectionRightsModel.has_reference_to_user_id`. (the same for `ExplorationRightsModel`)